README.md

```diff
@@ -0,0 +1,120 @@
+# pytest-cookies
+
+[pytest][pytest] is a mature testing framework for Python that is developed
+by a thriving and ever-growing community of volunteers. It uses plain assert
+statements and regular Python comparisons. At the core of the pytest test
+framework is a powerful hook-based plugin system.
+
+**pytest-cookies** is a pytest plugin that comes with a ``cookies`` fixture
+*which is a
+wrapper for the [cookiecutter][cookiecutter] API for generating projects. It
+helps you verify that your template is working as expected and takes care of
+cleaning up after running the tests. 🍪
+
+# Installation
+
+**pytest-cookies** is available for download from [PyPI][pypi] via [pip][pip]:
+
+```text
+pip install pytest-cookies
+```
+This will automatically install [pytest][pytest] and
+[cookiecutter][cookiecutter].
+
+# Usage
+
+## Generate a new project
+
+The ``cookies.bake()`` method generates a new project from your template
+based on the default values specified in ``cookiecutter.json``:
+
+```python
+def test_bake_project(cookies):
+    result = cookies.bake(extra_context={"repo_name": "helloworld"})
+
+    assert result.exit_code == 0
+    assert result.exception is None
+    assert result.project.basename == "helloworld"
+    assert result.project.isdir()
+```
+
+This also accepts the ``extra_context`` keyword argument that will be passed
```

Comment:
Maybe `cookies.bake()` or just `bake` instead of `This`?

Reply:
Good catch! Thank you 🙇